src/traits/election/vrf.rs

```diff
@@ -250,9 +250,7 @@ where
         let new_seed = *hasher.finalize().as_bytes();
         let mut prng = rand::rngs::StdRng::from_seed(new_seed);
 
-        #[allow(clippy::let_unit_value)]
-        let parameters = SIGSCHEME::param_gen(Some(&mut prng)).unwrap();
```

Comment:
So, uh... 1) this param_gen() apparently actually needed to be called for some `SignatureScheme` implementations (but not this one?) before calling `key_gen()`, in at least the version of jellyfish we were using when we wrote this, according to the documentation, but 2) this is basically dead code now. It was used for the sortition based VRF election scheme that turned out to be unsafe, and nothing uses this implementation of the election trait now. Even though we're likely to use a VRF based selection mechanism for DA committee membership in the future, it won't be this one.